pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -665,7 +676,10 @@ def status(self):
                 return _("Error setting up installation from HDD")
             return os.path.basename(self._currentIsoFile)
         elif self.payload.baseRepo:
-            return _("Closest mirror")
+            if self._payload_module.proxy.IsRedHatCDNEnabled:
+                return _("Red Hat CDN")
```

Comment:
I don't understand why `Red Hat CDN` is defined twice in this property.

Reply:
Good point, the second one is redundant, dropping it now.